test/udfs/test_yolo_object_detector.py

```diff
@@ -0,0 +1,74 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import sys
+import unittest
+
+import cv2
+import mock
+import pandas as pd
+
+from eva.models.storage.batch import Batch
+
+NUM_FRAMES = 10
```

Comment:
Can we get rid of FasterRCNN maybe in a separate PR?

Reply:
Yes Sir, we can do that.